server/index.js

```diff
@@ -0,0 +1,102 @@
+const path = require('path');
+const resolve = file => path.resolve(__dirname, file);
+const chokidar = require('chokidar');
+
+const express = require('express');
+const app = express();
+
+const PRODUCTION = process.env.NODE_ENV == 'production';
+
+if (!PRODUCTION) {
+  const dotenv = require('dotenv');
+  const loadEnv = envPath => {
+    try {
+      dotenv.config({ path: envPath });
+    } catch (err) {
+      // only ignore error if file is not found
+      if (err.toString().indexOf('ENOENT') < 0) {
+        throw err;
+      }
+    }
+  }
+
+  // Contrary to what you'd expect, the first env loaded is the one whose values are kept
+  loadEnv(resolve('../.env.local'));
+  loadEnv(resolve('../.env'));
+}
+
+const PORT = process.env.PORT;
+const HOST = process.env.HOST;
```

Comment:
I feel like something like INSTANCE_HOST/PORT would be better, unless naming the local server's address `HOST`/`PORT` is a .env convention.

Reply:
`HOST` and `PORT` are nodejs conventions

Reply:
👍